contracts/utils/DynamicBuffer.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2021 Divergent Technologies Ltd (github.com/divergencetech)
```

Comment:
Both work fine for me - I'd go with whichever you prefer.

I'm leaning a little more towards the latter one, because I think this will make things easier in the long run. I don't think that there are more implications other than a changed naming / attribution, since the license is MIT anyways. How do other projects handle this?

Reply:
Cool, I'll accept everything as is for now and then change to the AUTHORS approach in a different commit.